honeycombio/provider.go

```diff
@@ -21,14 +22,25 @@ func init() {
 // overwritten during the release process.
 var providerVersion = "dev"
 
+func deprecatedEnvCheck() schema.SchemaDefaultFunc {
+	if _, ok := os.LookupEnv("HONEYCOMBIO_APIKEY"); ok {
+		ctx := context.Background()
+		tflog.Warn(ctx, "HONEYCOMBIO_APIKEY is being depcrated, please use HONEYCOMB_API_KEY instead")
```

Comment:
Minor typo/phrasing fix, but this is great! ✨ 

```suggestion
		tflog.Warn(ctx, "HONEYCOMBIO_APIKEY has been deprecated, please use HONEYCOMB_API_KEY instead")
```

Reply:
Ahhh let me make change.

Reply:
Thank you! We'll get this into 0.9.0